packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -638,45 +636,39 @@ abstract class ObjectGroupBase implements Disposable {
     String methodName,
   ) async {
     if (disposed) return null;
-    if (useDaemonApi) {
-      return parseDiagnosticsNodeDaemon(invokeServiceMethodDaemon(methodName));
-    } else {
-      return parseDiagnosticsNodeObservatory(
-        invokeServiceMethodObservatory(methodName),
-      );
-    }
+    return useDaemonApi
+        ? parseDiagnosticsNodeDaemon(invokeServiceMethodDaemon(methodName))
+        : parseDiagnosticsNodeObservatory(
+            invokeServiceMethodObservatory(methodName),
+          );
   }
 
   Future<RemoteDiagnosticsNode?> invokeServiceMethodReturningNodeInspectorRef(
     String methodName,
     InspectorInstanceRef? ref,
   ) async {
     if (disposed) return null;
-    if (useDaemonApi) {
-      return parseDiagnosticsNodeDaemon(
-        invokeServiceMethodDaemonInspectorRef(methodName, ref),
-      );
-    } else {
-      return parseDiagnosticsNodeObservatory(
-        invokeServiceMethodObservatoryInspectorRef(methodName, ref),
-      );
-    }
+    return useDaemonApi
```

Comment:
this was a larger block but I still think this was a little bit of an improvement over the original code.